CMakeLists.txt

```diff
@@ -79,6 +79,7 @@ INCLUDE(${CMAKE_SOURCE_DIR}/cmake/set_default_flags.cmake)
 # numpy v1.7's documentation.
 add_definitions(-DNPY_NO_DEPRECATED_API=NPY_1_7_API_VERSION)
 
+set(NUMPY_INCLUDE_DIRS "" CACHE PATH "Additional NUMPY include path")
```

Comment:
You are setting it to empty string. Doesn't this remove what is coming from the command line?

Reply:
This sets the default if it is not in the cache. FORCE would override any preset value, but reading the doc on set(CACHE) again this is probably not the correct way to do this

Reply:
I am also not really sure why just setting the variable from the commandline with `-D` without the cache entry did not work

Reply:
Ok I am confused, it works perfectly fine without that line on ubuntu and yocto now.